src/App.js

```diff
@@ -1,21 +1,38 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-import ProductDetail from './components/ProductDetail';
- import footer from './components/footer.js';
-  import Header from './components/Header.js';
- import Carousel from './components/Carousel.js';
+import React, { Component } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import ProductDetail from "./components/ProductDetail";
+import footer from "./components/Footer.js";
```

Comment:
Double check this. Everything is case sensitive. What is the name of the actual file vs how you are importing it.